src/helpers/github.ts

```diff
@@ -0,0 +1,105 @@
+import fs from "fs";
+import got from "got";
+import makeDir from "make-dir";
+import promisePipe from "promisepipe";
+import tar from "tar";
+
+import packageJson from "../../package.json";
+import { isUrlOk } from "./networking";
+
+const ceaEnv: string = process.env.CEA_ENV || "";
+const githubApiBaseUrl: string = "https://codeload.github.com/" + packageJson.repository.name + "/tar.gz/";
+const semanticVersionRegex: RegExp = /(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?/
+
+console.log({
```

Comment:
We should remove this (or at least only print it if CEA_ENV is set)

Reply:
Of corso, I meant to remove them before committing.